### PR TITLE
GFORMS-1610 - Improve form versioning to handle existing accessibility statement for the forms

### PR DESCRIPTION
**Pull request by DursunGundogan**

### CODE REVIEW

**Review by rhubner — APPROVED** [review]

File: project/Dependencies.scala [hunk 1]
```diff
@@ -29,7 +29,8 @@ object Dependencies {
     "com.miguelfonseca.completely" % "completely-core"            % "0.8.0",
     "org.jsoup"                    % "jsoup"                      % "1.13.1",
     "org.webjars.npm"              % "govuk-frontend"             % "3.11.0",
-    "com.openhtmltopdf"            % "openhtmltopdf-pdfbox"       % "1.0.10"
+    "com.openhtmltopdf"            % "openhtmltopdf-pdfbox"       % "1.0.10",
+    "ai.x"                        %% "play-json-extensions"       % "0.42.0"
```
Comment (rhubner) [inline project/Dependencies.scala:33]: Do we need this exstension?

File: app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormTemplate.scala [hunk 2]
```diff
@@ -57,7 +60,7 @@ object FormTemplate {
   import JsonUtils._
 
   private val reads = Reads[FormTemplate] { json =>
-    Json.reads[FormTemplate].reads(json)
+    Jsonx.formatCaseClass[FormTemplate].reads(json)
```
Comment (rhubner) [inline app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormTemplate.scala:63]: Why are you using Jsonx?

**Review by rhubner — COMMENTED** [review]
Questions in code.

File: project/Dependencies.scala (hunk 1 above)
Comment (DursunGundogan) [reply project/Dependencies.scala:33]: `FormTemplate` case class has more than 22 members. This extension supports a format for case classes with more than 22 members.

File: app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormTemplate.scala (hunk 2 above)
Comment (DursunGundogan) [reply app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormTemplate.scala:63]: It allows us to read JSON format for case class with more than 22 members.

**Review by rhubner — APPROVED** [review]

**Pull request merged**